### PR TITLE
Historical graph

### PR DESCRIPTION
**Pull request by ReneMoraales**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ import SchoolPicker from './app/components/school-picker.vue';
 import YearPicker from './app/components/year-picker.vue';
 import LevelPicker from './app/components/level-picker.vue';
 import LevelYearPicker from './app/components/level-year-picker.vue';
-import Results from './app/components/results.vue';
+import ResultsColorMap from './app/components/results/color-map.vue';
+import ResultsHistorical from './app/components/results/historical.vue';
```
Comment (monkeyci) [inline app/assets/javascripts/application.js]: Unable to resolve path to module './app/components/results/historical.vue'     import/no-unresolved

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ import SchoolPicker from './app/components/school-picker.vue';
 import YearPicker from './app/components/year-picker.vue';
 import LevelPicker from './app/components/level-picker.vue';
 import LevelYearPicker from './app/components/level-year-picker.vue';
-import Results from './app/components/results.vue';
+import ResultsColorMap from './app/components/results/color-map.vue';
```
Comment (monkeyci) [inline app/assets/javascripts/application.js]: Unable to resolve path to module './app/components/results/color-map.vue'      import/no-unresolved

File: app/assets/javascripts/app/store/modules/results.js [hunk 3]
```diff
@@ -51,22 +52,27 @@ export default {
...
-          ResultsApi.getLevelResults(schoolId, context.state.level, context.state.year, indicatorIds).then(({ body: resultsBody }) => {
-            const results = resultsBody.results.map(result => {
-              result.school_id = schoolId;
+        YEARS_TO_FETCH.forEach(year => {
+          newResults[year] = [];
+
+          resultPromises.push(
+            ResultsApi.getLevelResults(schoolId, context.state.level, year, indicatorIds).then(({ body: resultsBody }) => {
+              const results = resultsBody.results.map(result => {
+                result.school_id = schoolId;
```
Comment (monkeyci) [inline app/assets/javascripts/app/store/modules/results.js]: Identifier 'school_id' is not in camel case              camelcase

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -25,7 +25,9 @@ import SchoolPicker from './app/components/school-picker.vue';
 import YearPicker from './app/components/year-picker.vue';
 import LevelPicker from './app/components/level-picker.vue';
 import LevelYearPicker from './app/components/level-year-picker.vue';
-import Results from './app/components/results.vue';
+import ResultsHeader from './app/components/results/results-header.vue';
+import ResultsColorMap from './app/components/results/color-map.vue';
+import ResultsHistorical from './app/components/results/historical.vue';
```
Comment (monkeyci) [inline app/assets/javascripts/application.js]: Unable to resolve path to module './app/components/results/historical.vue'      import/no-unresolved

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -25,7 +25,9 @@ import SchoolPicker from './app/components/school-picker.vue';
 import YearPicker from './app/components/year-picker.vue';
 import LevelPicker from './app/components/level-picker.vue';
 import LevelYearPicker from './app/components/level-year-picker.vue';
-import Results from './app/components/results.vue';
+import ResultsHeader from './app/components/results/results-header.vue';
+import ResultsColorMap from './app/components/results/color-map.vue';
```
Comment (monkeyci) [inline app/assets/javascripts/application.js]: Unable to resolve path to module './app/components/results/color-map.vue'       import/no-unresolved

File: app/assets/javascripts/application.js [hunk 6]
```diff
@@ -25,7 +25,9 @@ import SchoolPicker from './app/components/school-picker.vue';
 import YearPicker from './app/components/year-picker.vue';
 import LevelPicker from './app/components/level-picker.vue';
 import LevelYearPicker from './app/components/level-year-picker.vue';
-import Results from './app/components/results.vue';
+import ResultsHeader from './app/components/results/results-header.vue';
```
Comment (monkeyci) [inline app/assets/javascripts/application.js]: Unable to resolve path to module './app/components/results/results-header.vue'  import/no-unresolved

**Comment by llekn** [comment]
@ReneMoraales Se ve genial, salvo un pequeño detalle....

Al elegir colegios con nombres largos (casi todos), el selector de indicadores queda inutilizable:
image

Probemos poniendo el selector una línea más abajo que los colegios, y que siga alineado a la derecha?

**Pull request merged**